src/shared_fs.py

```diff
@@ -19,11 +20,19 @@
 from utilities import execute_command
 
 DIR_NO_MOUNTPOINT_EXIT_CODE = 32
+METRICS_STORAGE_USER_CONFIG = SystemUserConfig(
+    user="ubuntu",
+    group="ubuntu",
+)
```

Comment:
For LXD mode, we still use the ubuntu user for metrics storage to reduce the risk of breaking something. This can be changed in a follow-up PR.

Reply:
Please make sure we have a ticket for this

Reply:
Done